### DISCUSSION TITLE
Is it possible to declare css variable inside unocss config file, and use it with unocss?

### DISCUSSION THREAD

**Discussion by AxelBriche** [unocss/unocss#1789] [Q&A]
example: `mr-[--layout-gap]`

**Comment by zyyv** [comment]
You can add your css variables in `preflights`. And in UnoCSS, use the `$variable` syntax to use css variables.

e.g. [Playground](https://uno.antfu.me/play/?html=DwEwlgbgBAxgNgQwM5ILwCIC2AXAtAEkQE8B7AVzwHMEAHdAPgCgopsBTJbKGgJzYDM4YSgAtsSRsAD04CPSA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCqWEcAvnOlBCHAOQCujAYypVuAKFGoAHpFgoMAQ14AbeGkw58hYEQAUCUXHKV0S7QAsYVAFxwA2gcOIHjuEVq4Ayh50BKJy5dKGF4oLB4rDgh4JABaGKV5AE8IXhgYonkwGwBGAAZKLmYxAJYSZ1KHAF0ywwpqWms7Opp6Rl9q0WYfUSA&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwF8g)

**Reply by SKT1yang** [reply]
How to generate var(--layout-gap, 10px)

**Reply by zyyv** [reply]
Supported in https://github.com/unocss/unocss/pull/3453. Waiting for merge.
